doc/customizing/initialization-files.md

```diff
@@ -76,8 +76,8 @@ Catalog files can be edited two ways:
 |stories|no|[**Story[]**](#story)||An array of stories to be loaded.|
 |viewerMode|no|**"3d"** or **"3dSmooth"** or **"2D"** |"3d"|The id of the viewer mode to be shown initialy.|
 |homeCamera|yes|[**CameraPosition**](#CameraPosition)||Where the camera goes when you click the "home" button between the zoom-in and zoom-out buttons.|
-|<a id="base-maps"></a>baseMaps|no|[**`baseMaps`**](#baseMaps)|[default list of basemaps](../../lib/Models/BaseMaps/defaultBaseMaps.ts)|The array of the base maps to be shown to the user.|
-|<a id="base-map-id"></a>baseMapId|no|**string**||The id of the baseMap user will see on the first mapLoad. The value must be an id of the catalog item from the [`baseMaps`](#base-maps) array|
+|<a id="base-maps"></a>baseMaps|no|[**`baseMaps`**](#baseMaps)|[default list of basemaps](https://github.com/TerriaJS/terriajs/blob/next/lib/Models/BaseMaps/defaultBaseMaps.ts)|The array of the base maps to be shown to the user.|
```

Comment:
Have to use an absolute URL, hosted docs can't resolve relative URL to the repository.

Reply:
Good catch! We just need to remember to change the branch here once we have switched the branches.

Reply:
@na9da would you please submit a ticket and place it top of backlog so we don't forget when we switch to make this change?